src/app/planet/page.tsx

```diff
@@ -7,9 +7,9 @@ import { PlanetEnterButtons } from '~/modules/PlanetEnterButtons';
 const PlanetPage = () => {
   return (
     <div className="flex h-screen flex-col justify-center px-5">
-      <h1 className="mb-4 mt-1 text-h2 text-grey-900">아직 소속된 행성이 없네요!</h1>
+      <h1 className="mb-12pxr mt-4pxr text-h2 text-grey-900">아직 소속된 행성이 없네요!</h1>
       <p className="text-b2 text-grey-700">행성을 만들거나 초대된 행성으로 이동해보세요.</p>
-      <Image src="/assets/images/planet.png" alt="planet" width={400} height={400} />
+      <Image src="/assets/images/n-planet.png" alt="planet" width={400} height={400} />
```

Comment:
local로 돌렸을 때 이미지가 보이지 않는 현상이 있습니다 .
경로와 이미지 이름이 맞지 않았던 게 문제 같아요. 아래 수정 코드 제안드립니다!
```suggestion

import image from '../../../public/assets/images/planet.png';

const PlanetPage = () => {
  return (
    <div className="flex h-screen flex-col justify-center px-5">
      <h1 className="mb-12pxr mt-4pxr text-h2 text-grey-900">아직 소속된 행성이 없네요!</h1>
      <p className="text-b2 text-grey-700">행성을 만들거나 초대된 행성으로 이동해보세요.</p>
      <Image src={image} alt="planet" width={400} height={400} />
      <PlanetEnterButtons />
    </div>
  );
};
```

Reply:
이름이 잘못들어갔네요! 수정했습니다~~ 확인감사합니다!